doc/tutorials/acrn_on_qemu.rst

```diff
@@ -11,7 +11,7 @@ with basic functionality such as running Service VM (SOS) and User VM (UOS) for
 
 This setup was tested with the following configuration,
 
-- ACRN Hypervisor: tag acrn-2020w19.5-140000p (Commit ID: b0ae9cfa2b374d3faad9f890ff61e88f97205ac8)
+- ACRN Hypervisor: tag v2.0 (Not support in later version)
```

Comment:
```suggestion
- ACRN Hypervisor: tag ``v2.0``
```

Reply:
Then add a note below this bullet list using the `.. note::` directive. I would suggest the following wording: "ACRN versions newer than ``v2.0`` do not work on QEMU."

Reply:
Updated.